scripts/smart-status.sh

```
@@ -75,7 +76,7 @@ if [ -n "$_jq_version" ]; then
   _jq_major="${_jq_version%%.*}"
   _jq_minor="${_jq_version#*.}"; _jq_minor="${_jq_minor%%.*}"
   if [ "${_jq_major:-0}" -lt 1 ] || { [ "${_jq_major:-0}" -eq 1 ] && [ "${_jq_minor:-0}" -lt 6 ]; }; then
-    echo "Warning: jq $_jq_version detected — staleness features require jq 1.6+. Team Activity may not display." >&2
+    echo "Warning: jq $_jq_version detected â€” staleness features require jq 1.6+. Team Activity may not display." >&2
```

Shellcheck (0.11.0)

[warning] This is a unicode quote. Delete and retype it (or ignore/singlequote for literal).

(SC1111)